pkg/api/output.go

```diff
@@ -214,6 +234,49 @@ func (instance *Instance) markdownOut(columns columnList) *tablewriter.Table {
 	return table
 }
 
+func (instance *Instance) csvOut(columns columnList) (*csv.Writer, error) {
+	csvWriter := csv.NewWriter(os.Stdout)
+
+	if len(instance.Outputs) == 0 {
+		_, _ = fmt.Println("No output to display")
+	}
+
+	columnIndexes := make([]int, 0, len(columns))
+	for k := range columns {
+		columnIndexes = append(columnIndexes, k)
+	}
+	sort.Ints(columnIndexes)
+
+	var csvData [][]string
+
+	var headers []string
+	for _, k := range columnIndexes {
+		headers = append(headers, columns[k].header())
+	}
+
+	csvData = append(csvData, headers)
+
+	for _, o := range instance.Outputs {
+		var row []string
+		for _, k := range columnIndexes {
+			if columns[k].value(o) == "" {
+				row = append(row, "n/a")
```

Comment:
This might be uncalled for when thinking about it, during testing I noticed that I got an extra trailing `,` in the end of rows (last column) where there are no data (empty `string`). I then added something to replace that with, in this case a `n/a`, to fill in the blanks basically.

I don't think that the trailing `,` is a problem but adding `n/a` might be something users won't expect since there are no other output options that add this to the output of data. This would also add an `n/a` at any place where there is an empty `string`.

WDYT?

Reply:
I think you're right that a trailing comma is normal in a csv file and in fact ok if there are empty columns with a few commas in a row having nothing between them. We can remove that `n/a` as you suggest.

Reply:
Great! Fixed in: https://github.com/FairwindsOps/pluto/pull/279/commits/64edde6d505b5d16539be20c45f6ff8cda66e719